files/install-deps-worker.yaml

```
@@ -22,11 +22,26 @@
           - python3-bugzilla # python-bugzilla (not bugzilla) on PyPI
           - python3-backoff # Bugzilla class
           #- python3-flask-restx # needs Fedora 32
+          - dnf-utils
+          - python3-pip
+          - make
         state: present
+    - name: Download packit spec file
+      get_url:
+        url: https://raw.githubusercontent.com/packit-service/packit/master/packit.spec
+        # this should be improved - zuul is using var peoject_dir - can be used also in non-zuul
+        # runs - if not set -> set?, will be there any zuul runs conflicts?
+        dest: ./packit.spec
+    - name: Build packit RPM dependencies based on packit.spec
```

```suggestion
    - name: Install packit RPM build dependencies
```

But I think we said the `BuildRequires` are not enough and we want to install `Requires`. @csomh suggested:
```sh
dnf install $(rpmspec -q --requires packit.spec | grep -v 'packit')
```

---

fixed